src/sympc/api.py

```diff
@@ -227,6 +227,14 @@
         "sympc.tensor.replicatedshare_tensor.ReplicatedSharedTensor.numel",
         "syft.lib.python.Int",  # FIXME: Can't we just return an int??
     ),
+    (
+        "sympc.tensor.replicatedshare_tensor.ReplicatedSharedTensor.get_ring_size",
```

Comment:
MAYBE: This one should be added to the ReplicatedShareTensor?

Reply:
Crct.It here george: 
https://github.com/OpenMined/SyMPC/blob/113895e1e15b253b2e7467a05cebf0f2876e139d/src/sympc/tensor/replicatedshare_tensor.py#L146